publications-2019.bib

```diff
@@ -1,5 +1,222 @@
+  title = {Bridging the terahertz gap for chaotic sources with superconducting junctions},
+  journal = {Physical Review B}
+}
+
+@article{German2019,
+  doi = {10.1016/j.anucene.2019.05.049},
+  url = {https://doi.org/10.1016/j.anucene.2019.05.049},
+  year = {2019},
+  month = dec,
+  publisher = {Elsevier {BV}},
+  volume = {134},
+  pages = {144--157},
+  author = {P{\'{e}}ter German and Jean C. Ragusa},
+  title = {Reduced-order modeling of parameterized multi-group diffusion k-eigenvalue problems},
+  journal = {Annals of Nuclear Energy}
+}
+
+@phdthesis{gerken2019dynamic,
+  title={Dynamic Inverse Problems for Wave Phenomena},
+  author={Gerken, Thies},
+  year={2019},
+  school={Universit{\"a}t Bremen}
+}
+
+@article{https://doi.org/10.13140/rg.2.2.18015.41123,
+  doi = {10.13140/RG.2.2.18015.41123},
+  url = {http://rgdoi.net/10.13140/RG.2.2.18015.41123},
+  author = {Exner,  Pavel},
+  language = {en},
+  title = {Extended finite element method for approximation of singularities in groundwater flow},
+  publisher = {Unpublished},
+  year = {2019}
+}
+
+@article{Deolmi2019,
+  doi = {10.1016/j.amc.2018.11.059},
+  url = {https://doi.org/10.1016/j.amc.2018.11.059},
+  year = {2019},
+  month = may,
+  publisher = {Elsevier {BV}},
+  volume = {348},
+  pages = {234--256},
+  author = {G. Deolmi and S. M\"{u}ller and M. Albers and P.S. Meysonnat and W. Schr\"{o}der},
+  title = {A reduced order model to simulate compressible flows over an actuated riblet surface},
+  journal = {Applied Mathematics and Computation}
+}
+
+@article{Vassaux2019,
+  doi = {10.1002/adts.201800168},
+  url = {https://doi.org/10.1002/adts.201800168},
+  year = {2019},
+  month = feb,
+  publisher = {Wiley},
+  volume = {2},
+  number = {5},
+  pages = {1800168},
+  author = {Maxime Vassaux and Robert C. Sinclair and Robin A. Richardson and James L. Suter and Peter V. Coveney},
+  title = {The Role of Graphene in Enhancing the Material Properties of Thermosetting Polymers},
+  journal = {Advanced Theory and Simulations}
+}
+
+@article{Freddi2019,
+  doi = {10.1016/j.mechrescom.2019.01.009},
+  url = {https://doi.org/10.1016/j.mechrescom.2019.01.009},
+  year = {2019},
+  month = mar,
+  publisher = {Elsevier {BV}},
+  volume = {96},
+  pages = {29--36},
+  author = {Francesco Freddi},
+  title = {Fracture energy in phase field models},
+  journal = {Mechanics Research Communications}
+}
+
+@article{Bzowski2019,
+  doi = {10.1088/1757-899x/627/1/012018},
+  url = {https://doi.org/10.1088/1757-899x/627/1/012018},
+  year = {2019},
+  month = oct,
+  publisher = {{IOP} Publishing},
+  volume = {627},
+  pages = {012018},
+  author = {Krzysztof Bzowski and Lukasz Rauch and Maciej Pietrzyk},
+  title = {Selection of the optimal heat treatment conditions using advanced phase transformation models},
+  journal = {{IOP} Conference Series: Materials Science and Engineering}
+}
+
+@article{https://doi.org/10.11588/heidok.00027466,
```

Comment:
Is this a valid bibtex key?

Reply:
Also, this is a dissertation, not an article, according to the website.
How can an dissertation have two authors?!

Reply:
> How can an dissertation have two authors?!

Ah, okay. It's just first name, middle name, and then last name that consists of two words. It's okay then, I got it wrong :-)